operatorcourier/errors.py

```diff
@@ -0,0 +1,46 @@
+"""
+operatorcourier.errors
+
+Defines custom operator-courier exceptions.
+"""
+
+
+class OpCourierError(ValueError):
```

Comment:
Could we add unit test that `OpCourierError` (and children) are subclass of `ValueError` to make sure we will keep this BW compatibility. Unless @kevinrizza agree to break compatibility

Reply:
Maintaining backwards compatibility is a major goal in the short term, since we have only recently released. Making breaking changes should be reserved for a specific release goal, which this pr is not aiming for. Please maintain backwards compatibility.